redis/gke-redis/variables.tf

```diff
@@ -0,0 +1,30 @@
+variable "project" {
+    type        ="string"
+}
+variable "name" {
+    type = "string"
+}
+variable "region" {
+    type        = "string"
+}
+variable "zone" {
```

Comment:
If I recall correctly `Terraform` throws a warning to have this changed to `location` since `zone` has been/will be deprecated.